trajectory_msgs/JointTrajectoryPoint Init
trajectory_msgs/JointTrajectoryPoint Goal
---
bool status
trajectory_msgs/JointTrajectory plan
